src/deepqmc/conf/ansatz/default.yaml

```
@@ -7,6 +7,7 @@ envelope:
   per_shell: false
   per_orbital_exponent: true
   spin_restricted: false
+  init_to_ones: false
```

```suggestion
  init_to_ones: true
```
Don't we want to init to ones whenever possible, since it helps so much?